pkg/expression/integration_test/integration_test.go

```diff
@@ -63,7 +63,7 @@ func TestGetLock(t *testing.T) {
 	// Increase pessimistic txn max retry count to make test more stable.
 	originCfg := config.GetGlobalConfig()
 	newCfg := *originCfg
-	newCfg.PessimisticTxn.MaxRetryCount = 2048
+	newCfg.PessimisticTxn.MaxRetryCount = 10240
```

Comment:
Why change this value?

Reply:
As `Increase pessimistic txn max retry count to make test more stable.`, but sometimes `2048` is not big enough.